### ISSUE TITLE
Handle exabyte and more

### ISSUE THREAD

**Issue by clintnetwork** [omar/ByteSize#54]
hey

@omar @jetersen 

Can you provide exabyte and more?

**Comment by jetersen** [comment]
@clintnetwork I am only a contributor.

I suggest you open a pull request to add exabyte

**Comment by clintnetwork** [comment]
If I do pull request it will be merged enough fastly?

@omar

**Comment by omar** [comment]
@clintnetwork yes, I should have time to dedicate to the code review over the coming weeks.

**Comment by clintnetwork** [comment]
Okey because exabyte support would be great and there a bug with big int too
